app/lib/actions/katello/repository/update.rb

```diff
@@ -35,6 +34,15 @@ def plan(root, repo_params)
                         :type => root.content_type,
                         :arches => root.format_arches)
 
+            if update_auto_enabled
```

Comment:
should this content update logic go into a new action like Katello::Product::ContentUpdate ?